community/CONTRIBUTING.md

```diff
@@ -5,6 +5,195 @@ Thanks for considering contributing to OPEA project. The contribution process is
+│   │   └── hpu
+│   │       └── gaudi
+│   │           ├── compose.yaml
+│   │           ├── how_to_validate_service.md    #optional
+│   │           ├── README.md
+│   │           └── set_env.sh
+│   └── nvidia
+│       └── gpu   
+│           ├── compose.yaml
+│           ├── README.md
+│           └── set_env.sh
+├── Dockerfile
+├── docker_image_build
+│   └── build.yaml
+├── kubernetes
+│   ├── intel
+│   │   ├── cpu
+│   │   │   └── xeon
+│   │   │       ├── gmc
+│   │   │       │   └── chatQnA_xeon.yaml
+│   │   │       └── manifest
+│   │   │           └── chatqna.yaml     # could be multiple .yaml for various applications
+│   │   └── hpu
+│   │       └── gaudi
+│   │           ├── gmc
+│   │           │   └── chatQnA_gaudi.yaml
+│   │           └── manifest
+│   │               └── chatqna.yaml
+│   ├── amd
+│   │   ├── cpu      
+│   │   │   ├── gmc
+│   │   │   └── manifest
+│   │   └── gpu
+│   │       ├── gmc
+│   │       └── manifest
+│   ├── README_gmc.md  # K8s quickstar
+│   └── README.md      # quickstart
+├── README.md
+├── tests
+│   ├── test_compose_on_gaudi.sh  #could be more tests for different flavors of the app
+│   ├── test_gmc_on_gaudi.sh
+│   ├── test_manifest_on_gaudi.sh
+└── ui
+
+```
+- **File Descriptions**:
+  - `chatqna.py`: application definition using microservice, megaservice and gateway. There could be multiple .py in the folder based on slight modification of the example application.
+  - `docker_build_image/build.yaml`: builds necessary images pointing to the Dockerfiles in the GenAIComp repository. 
+  - `docker_compose/vendor/device/compose.yaml`: defines pipeline for  docker compose deployment. For selectng docker image name please follow the naming convention: 
+      - Docker Image: `opea/[example name]-[feature name]:latest` all lower case (i,e: opea/chatqna, opea/codegen-react-ui)
+  - `kubernetes/vendor/device/manifests/chatqna.yaml`: used for K8s deployemnt
+  - `kubernetes/vendor/device/gmc/chatqna.yaml`: (optional) used for deployment with GMC
+  - `tests/`: at minimum you need to provide an E2E test with docker compose. If you are contritbutng K8s manifests and GMC yaml, you should also provide test for those. Please follow naming convention: 
+    - docker compose test: `tests/test_compose_on_[hardware].sh`
+    - K8s test: `tests/test_manifest_on_[hardware].sh`
+    - K8s with GMC test: `tests/test_gmc_on_[hardware].sh`
+  - `ui`: (optional)
+  - `assets`: nice to have an application flow diagram
+
+#### Additional steps if your contribution is Hardware Specific
+
+You will need additional step to configure CI/CD for merging your GenAIComp or GenAIExample. 
+- Connect hardware into OPEA GHA as self-hosted runner 
+- Contribute new test scripts for the new hardware:
+- Dockerfile for the Component (i,e `GenAIComp/comps/llm/text-generation/tgi/docker/Dockerfile_[vendor]_[hardware]` )
+- Update image build yaml for new images​
+- Update CI/CD workflow to identify and deploy new test
```

Comment:
We can remove this line, after reorg, new CI/CD structure should adapt to most of the test case.

Reply:
@chensuyue do I use your email contact or something else?

Reply:
Have reached out for help to create an alias for future use -  to spread the load as the community grows and to ensure coverage when folks are on vacation.